src/libs/OptionsListUtils.ts

```diff
@@ -2491,6 +2512,15 @@ function filterOptions(options: Options, searchInputValue: string, config?: Filt
                 values = values.concat(getParticipantsLoginsArray(item));
             }
 
+            // We don't want the following to be searchable by participant:
+            // - Default rooms
+            // - Policy rooms
+            // - Invoice rooms
+            // - Trip rooms
```

Comment:
NAB: I feel like this comment makes more sense in the `isSearchableViaParticipants` function itself.

Reply:
I think we can actually just omit this comment altogether, I think isSearchableViaParticipants is pretty self-documenting.